queue/src/fiskaltrust.Interface.Tagging/DE/CaseConverterDE.cs

```diff
@@ -0,0 +1,494 @@
+﻿using fiskaltrust.Interface.Tagging.Interfaces;
+using fiskaltrust.ifPOS.v1;
+using V1 = fiskaltrust.Interface.Tagging.Models.V1;
+using V2 = fiskaltrust.Interface.Tagging.Models.V2;
+using fiskaltrust.Interface.Tagging.Models.Extensions;
+using fiskaltrust.Interface.Tagging.Models.V2.Extensions;
+using fiskaltrust.Interface.Tagging.Models.V1.DE.Extensions;
+
+namespace fiskaltrust.Interface.Tagging.DE
+{
+    public class CaseConverterDE : ICaseConverter
+    {
+        public void ConvertftChargeItemCaseToV1(ChargeItem chargeItem)
+        {
+            var v2ChargeItem = new ChargeItem() { ftChargeItemCase = chargeItem.ftChargeItemCase };
+
+            chargeItem.ftChargeItemCase = (long) ((ulong) v2ChargeItem.ftChargeItemCase & 0xFFFF_0000_0000_0000);
```

Comment:
Should we add a "CaseName" parameter to enums and use GetCaseName Extention? to avoid bitwise operation